org/lateralgm/file/GmFileReader.java

```diff
@@ -1027,7 +1027,10 @@ private static void readRooms(ProjectFileContext c) throws IOException,GmFormatE
 			rm.put(PRoom.SPEED,in.read4());
 			rm.put(PRoom.PERSISTENT,in.readBool());
 			rm.put(PRoom.BACKGROUND_COLOR,Util.convertGmColor(in.read4()));
-			rm.put(PRoom.DRAW_BACKGROUND_COLOR,in.readBool());
+			//NOTE: Mike Dailly must have done this lol, how does he not know what is conventionally true?
```

Comment:
He probably Googles his name... let's not.

Reply:
Alright fine, we can let him off the hook this one time: 46c7d904c8fe959f310713743b850d823884378b